libswirl/deps/lua/Makefile

```diff
@@ -0,0 +1,197 @@
+# Makefile for building Lua
```

Comment:
Do we actually use this makefile or is it part of the 3rdparty drop?

Reply:
Part of the 3rdparty package. Had to remove other things, but forgot to remove this file..